### PR TITLE
v3 - webpack 5

### PR DESCRIPTION
**Pull request by geoctrl**
# v3

### updates

- remove unneeded deps
- update rest of deps

### breaking changes

- compatibility with webpack 5 (uses new `this.getOptions()` method)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -7,7 +6,7 @@ let globalId = 0;
 module.exports = function(source) {
   const id = globalId;
   globalId += 1;
-  const loaderOptions = getOptions(this) || {};
+  const loaderOptions = this.getOptions() || {};
```
Comment (rhys-childs) [inline index.js:9]: am I wrong, or is this.getOptions() not defined so loaderOptions would always be {}?

Comment (geoctrl) [reply index.js:9]: https://webpack.js.org/api/loaders/#thisgetoptionsschema

`getOptions` is available in context - it's a method already set on the parent class

check out the code example here:
https://webpack.js.org/contribute/writing-a-loader/#loader-utilities

**Review by rhys-childs — APPROVED** [review]
Looks great!

File: index.js (hunk 1 above)
Comment (rhys-childs) [reply index.js:9]: Thanks! I figured I had to be missing something!

**Pull request merged**